datalake/backend/tests/test_processing_stack.py

```diff
@@ -0,0 +1,133 @@
+import json
+import logging
+import os
+import time
+
+from pytest import mark
+
+from .utils import S3Object, any_safe_file_path
+
+ENV = os.environ.get("DEPLOY_ENV", "dev")
+
+APPLICATION_TAG_KEY = "ApplicationName"
+APPLICATION_TAG_VAL = "geospatial-data-lake"
+
+ENVIRONMENT_TAG_KEY = "EnvironmentType"
+ENVIRONMENT_TAG_VAL = ENV
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class NoStateMachineFound(Exception):
+    pass
+
+
+class NoComputeEnvironmentFound(Exception):
+    pass
+
+
+def get_compute_environment(batch_client):
+    compute_environment_detection_response = batch_client.describe_compute_environments()
+
+    # make sure there are no more records in next pages
+    assert compute_environment_detection_response.get("nextToken", None) is None
+
+    for compute_environment in compute_environment_detection_response["computeEnvironments"]:
+        if (
+            compute_environment["tags"][ENVIRONMENT_TAG_KEY] == ENVIRONMENT_TAG_VAL
+            and compute_environment["tags"].get(APPLICATION_TAG_KEY, None) == APPLICATION_TAG_VAL
+        ):
+            datalake_compute_environment = compute_environment
+            logger.info("Datalake Compute Environment: %s", datalake_compute_environment)
+
+            return datalake_compute_environment
+
+    raise NoStateMachineFound(APPLICATION_TAG_VAL, ENVIRONMENT_TAG_VAL)
+
+
+def launch_compute_environment_resources(batch_client, datalake_compute_environment):
+    """
+    Trigger immediate launch of computing resources. This will significantly speed up Batch jobs
+    execution.
+    """
+    if (
+        int(datalake_compute_environment["computeResources"]["minvCpus"]) < 1
+        or int(datalake_compute_environment["computeResources"]["desiredvCpus"]) < 1
+    ):
+        batch_client.update_compute_environment(
+            computeEnvironment=datalake_compute_environment["computeEnvironmentName"],
+            computeResources={"minvCpus": 1, "desiredvCpus": 1},
+        )
+
+
+def get_state_machine(stepfunctions_client):
+    state_machines_detection_response = stepfunctions_client.list_state_machines()
+
+    # make sure there are no more records in next pages
+    assert state_machines_detection_response.get("nextToken", None) is None
```

Comment:
`.get("nextToken", None)` can be simplified to `.get("nextToken")`.

Reply:
Done in eaaca6eaf34b053597b534f771e4d88f91d3de6d